packages/consent-proof-signature/src/index.ts

```diff
@@ -13,7 +13,7 @@ export const createConsentMessage = (
   "XMTP : Grant inbox consent to sender\n" +
   "\n" +
   `Current Time: ${new Date(timestampMs).toUTCString()}\n` +
-  `From Address: ${peerAddress}\n` +
+  `Sender Address: ${peerAddress}\n` +
```

Comment:
This will need to be changed in:
iOS: https://github.com/xmtp/xmtp-ios/blob/main/Sources/XMTPiOS/Messages/Signature.swift#L77
Android: https://github.com/xmtp/xmtp-android/blob/main/library/src/main/java/org/xmtp/android/library/messages/Signature.kt#L56
After those there needs to be a React Native update to include those changes and update the tests to include that change

Reply:
Ah, bummer -- thanks for the heads-up. IMO that's currently not worth the effort for this change so I'm going to delete this PR.